openml/study/functions.py

```diff
@@ -277,7 +277,7 @@ def update_study_status(study_id: int, status: str) -> None:
     legal_status = {"active", "deactivated"}
     if status not in legal_status:
         raise ValueError("Illegal status value. " "Legal values: %s" % legal_status)
-    data = {"study_id": study_id, "status": status}
+    data = {"study_id": study_id, "status": status}  # type: Dict[str, Union[str, int]]
```

Comment:
Should this not be `# type: openml._api_calls.DATA_TYPE`?

Reply:
Good catch.